README.md

```diff
@@ -29,6 +29,19 @@ Update `prometheus/prometheus.yml` with the targets (server you wish to monitor)
 ./start-all.sh
 ```
 
+### Load original data to prometheus server
```

Comment:
@kongove can you explain where this data comes from?
Is it the same data capture in this procedure:
https://github.com/scylladb/scylla/wiki/How-to-report-a-Scylla-problem#prometheus